src/pages/quizzes.tsx

```diff
@@ -0,0 +1,233 @@
+    USER_STATS_KEY,
+    INITIAL_USER_STATS
+  )
+  const [isModalOpen, setModalOpen] = useState(false)
+
+  const { t } = useTranslation()
+  const heroContent = {
+    title: <Translation id="quizzes-title" />,
+    header: <Translation id="test-your-knowledge" />,
+    subtitle: <Translation id="quizzes-subtitle" />,
+    image: getImage(data.doge)!,
+    alt: t("quizzes-title"),
+  }
+
+  const contextState = {
+    status: quizStatus,
+    next: nextQuiz,
+    score: userStats.score,
+    quizKey: currentQuiz,
+    average: userStats.average,
+    completed: userStats.completed,
+    setUserStats: setUserStats,
+  }
+
+  return (
+    <Box>
+      <PageMetadata
+        title={t("quizzes-title")}
+        description={t("quizzes-subtitle")}
+      />
+      <Box w="100%" bg="layer2Gradient" pb={8}>
+        <PageHero content={heroContent} isReverse />
+      </Box>
+
+      <QuizzesHubContext.Provider value={contextState}>
+        <QuizzesModal isOpen={isModalOpen} setIsOpen={setModalOpen}>
+          <QuizWidget
+            quizKey={currentQuiz}
+            nextHandler={setCurrentQuiz}
+            statusHandler={setQuizStatus}
+            isStandaloneQuiz={false}
+          />
+        </QuizzesModal>
+
+        <Box px={{ base: 0, lg: 8 }} py={{ base: 0, lg: 4 }} mb={12}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            alignItems="start"
+            gap={{ base: 0, lg: 20 }}
+          >
+            {/* quizzes list */}
+            <Box flex={1} order={{ base: 2, lg: 1 }}>
+              <Box px={{ base: 8, lg: 0 }}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="basics" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="basics-description" />
+                </Text>
+
+                <QuizzesList
+                  content={ethereumBasicsQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Box px={{ base: 8, lg: 0 }} mb={10}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="using-ethereum" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="using-ethereum-description" />
+                </Text>
+
+                <QuizzesList
+                  content={usingEthereumQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Flex
+                direction={{ base: "column", xl: "row" }}
```

Comment:
```suggestion

              // Todo: Revert to md: "row" when we merge multi-line button support
              // Issue: https://github.com/ethereum/ethereum-org-website/issues/10020
              <Flex
                direction={{ base: "column", xl: "row" }}
```

@nhsz noticing in [this commit](https://github.com/ethereum/ethereum-org-website/pull/10320/commits/3d74a344fc7e5b918a5431eb81851feed4dbf806), we had to change the layout to fix the button overflow. 

We should revert this change once adding multiline button support (#10020) is resolved. cc: @corwintines @pettinarip

Reply:
@minimalsm I created a card to resume this when the quoted issue is solved, closing this